### PR TITLE
Correct panel buttons and methods

### PR DESCRIPTION
**Pull request by LoveMyDaughter**
The Admin panel has 4 buttons (Users, Owners, Moderators, Restaurants). In our tests, we'll use only the Moderators button for the admin role, that's why the only ClickModeratorsButton method is present here.

### CODE REVIEW

File: TAF_EasyRest/TestFramework/PageComponents/AdminPanelPageComponent.cs [hunk 1]
```diff
@@ -8,30 +8,19 @@ public class AdminPanelPageComponent : BasePage
 
         private IWebElement _usersButton => driver.FindElement(By.XPath("//span[text() = 'Users']/ancestor::a"));
         private IWebElement _ownersButton => driver.FindElement(By.XPath("//span[text() = 'Owners']/ancestor::a"));
+        private IWebElement _moderatorsButton => driver.FindElement(By.XPath("//span[text() = 'Moderators']/ancestor::a"));
```
Comment (andriifromlviv) [inline TAF_EasyRest/TestFramework/PageComponents/AdminPanelPageComponent.cs:11]: your XPath is working well but can be simplified to just "//span[text() = 'Moderators']"

**Review by andriifromlviv — DISMISSED** [review]

File: TAF_EasyRest/TestFramework/PageComponents/AdminPanelPageComponent.cs (hunk 1 above)
Comment (LoveMyDaughter) [reply TAF_EasyRest/TestFramework/PageComponents/AdminPanelPageComponent.cs:11]: Right!

File: TAF_EasyRest/TestFramework/PageComponents/AdminPanelPageComponent.cs [hunk 2]
```diff
@@ -8,30 +8,19 @@ public class AdminPanelPageComponent : BasePage
...
         private IWebElement _restaurantsButton => driver.FindElement(By.XPath("//span[text() = 'Restaurants']/ancestor::a"));
 
         #endregion
 
 
         #region Methods
 
-        public AdminPanelPageComponent ClickUsersButton()
+         public AdminPanelModeratorsPage ClickModeratorsButton()
```
Comment (dmatsev) [inline TAF_EasyRest/TestFramework/PageComponents/AdminPanelPageComponent.cs:19]: please remove 1 (one ) useless space before `public`
image

**Review by dmatsev — CHANGES_REQUESTED** [review]
Please resolve the comments.

File: TAF_EasyRest/TestFramework/PageComponents/AdminPanelPageComponent.cs (hunk 2 above)
Comment (LoveMyDaughter) [reply TAF_EasyRest/TestFramework/PageComponents/AdminPanelPageComponent.cs:19]: Done.

File: TAF_EasyRest/TestFramework/PageComponents/AdminPanelPageComponent.cs (hunk 1 above)
Comment (dmatsev) [reply TAF_EasyRest/TestFramework/PageComponents/AdminPanelPageComponent.cs:11]: @LoveMyDaughter, I don't see that you've resolved Andrii's comment. Are you agree with him or not?

Comment (LoveMyDaughter) [reply TAF_EasyRest/TestFramework/PageComponents/AdminPanelPageComponent.cs:11]: I want to leave ancestor::a inside the XPath - it's more clear what element we are looking for: "a" is a hyperlink.

**Review by dmatsev — APPROVED** [review]

**Review by sofiaso — APPROVED** [review]

**Pull request merged**